### PR TITLE
modified starting page as per wireframes

### PR DESCRIPTION
**Pull request by AnilMula**
# Description

This PR is change the existing root page to the one in wireframes

Fixes # (issue)

# How to test?

Tested in chrome browser of local machines

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by claudion3** [comment]
Hello @AnilMula  could you please add the issue number or link to the issue, because it help someone who going to review this PR to know more about the issue you resolved.

**Comment by AnilMula** [comment]
Hi @claudion3 there is no issue for that.

File: src/client/containers/Home/Home.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 import React from 'react';
+import Login from '../LoginPage/LoginPage.component';
 
-export const Home = () => <section className="home">Final project!</section>;
+export const Home = () => <Login />;
```
Comment (tpetrina) [inline src/client/containers/Home/Home.js:4]: Please remove this line - Home page will not host Login but will instead host basic information.

File: src/client/components/Navigation/Header.js [hunk 2]
```diff
@@ -26,24 +26,7 @@ export default function Header({ isAuthenticated = false }) {
...
-        <li>
-          <Link to="/sign-up">Sign up</Link>
-        </li>
-        <li>
-          <Link to="/reset-password">Reset password</Link>
-        </li>
-      </ul>
-    </nav>
-  );
+  return <div />;
```
Comment (tpetrina) [inline src/client/components/Navigation/Header.js:29]: What about the authenticated part? You should remove that too.
Simply return `null` in general.

**Review by tpetrina — CHANGES_REQUESTED** [review]

File: src/client/components/Navigation/Header.js (hunk 2 above)
Comment (AnilMula) [reply src/client/components/Navigation/Header.js:29]: corrected

File: src/client/containers/Home/Home.js (hunk 1 above)
Comment (AnilMula) [reply src/client/containers/Home/Home.js:4]: I was told like this: As per wireframes there is no home page, starting page is login page. Is it not true?

Comment (tpetrina) [reply src/client/containers/Home/Home.js:4]: The starting page can be seen in Figma after the vertical line in each row. Login is a separate page and only available when logging in.

**Review by tpetrina — CHANGES_REQUESTED** [review]

**Comment by tpetrina** [comment]
This PR is not necessary - will close it and delete it.

**Pull request closed without merging**